### PR TITLE
HTCONDOR-1694-remove-vestigal-docs

### PR DESCRIPTION
**Pull request by GregThain**
<Insert PR description here, and leave checklist below for code review.>

# HTCondor Pull Request Checklist for internal reviewers

- [x] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
- [x] Verify that the related Jira ticket exists and has a target version number and that it is correct.
- [x] Verify that the Jira ticket is in review status and is assigned to the reviewer.
- [x] Verify that the Jira ticket (HTCONDOR-xxx) is mentioned at the beginning of the title. Edit it, if not
- [x] Verify that the branch destination of the PR matches the target version of the ticket
- [ ] Check for correctness of change
- [ ] Check for regression test(s) of new features and bugfixes (if the feature doesn't require root)
- [x] Check for documentation, if needed
- [ ] Check for version history, if needed
- [ ] Check BaTLab dashboard for successful build (https://batlab.chtc.wisc.edu/results/workspace.php) and test for either the PR or a workspace build by the developer that has the Jira ticket as a comment.
- [x] Check that each commit message references the Jira ticket (HTCONDOR-xxx)

## After the above
- Hit the merge button if the pull request is approved and it is not a security patch (security changes require 2 additional reviews)
- If the pull request is approved, take the ticket out of review state
- Assign JIRA Ticket back to the developer

### CODE REVIEW

File: docs/admin-manual/introduction-admin-manual.rst [hunk 1]
```diff
@@ -5,17 +5,17 @@ This is the HTCondor Administrator's Manual. Its purpose is to aid in
...
-Every part of HTCondor sends periodic updates to the central manager,
-the centralized repository of information about the state of the pool.
-Periodically, the central manager assesses the current state of the pool
-and tries to match pending requests with the appropriate resources.
+An HTCondor pool :index:`pool<single: pool; HTCondor>`\ :index:`pool of
+machines` is comprised of a single machine which serves as the central manager,
+:index:`central manager`\ and an arbitrary number of other machines.  Machines
+intended to run work are called Execution Points (EP)s, also known as worker
+nodes.  Machines that hold a queue of jobs ready to run, or the results of jobs
+that have run are called Access Points (AP)s, also known as submit machines.
```
[the remark this answers is not in the document]
Comment (ColeBollig) [reply docs/admin-manual/introduction-admin-manual.rst:13]: I figured that was the case. Do we feel that the wording should be phrased in a way that keeps the association to reduce confusion but also implies that is old terminology by replacing "also known" with "formerly known". If you disagree I am content with the current phrasing

File: docs/admin-manual/configuration-macros.rst [hunk 2]
```diff
@@ -6532,22 +6508,6 @@ do not specify their own with:
...
-    be used.
-
-:macro-def:`DEFAULT_IO_BUFFER_BLOCK_SIZE`
-    When buffering is enabled, HTCondor will attempt to consolidate
-    small read and write operations into large blocks. This macro
-    specifies the default block size HTCondor will use. The
-    *condor_status* ``buffer_block_size`` command will override this
-    default. If this macro is undefined, a default size of 32 KB will be
-    used.
-
```
Comment (ColeBollig) [inline docs/admin-manual/configuration-macros.rst:6550]: A quick git grep shows that these are still being used in submit_utils.cpp

Comment (GregThain) [reply docs/admin-manual/configuration-macros.rst:6550]: Yes, but the classad attributes aren't being used for anything.  Eventually we'll rip that code out, but for now, no need to confuse user with extra info they don't need.

**Review by ColeBollig — APPROVED** [review]
LGTM

**Pull request merged**